lib/charms/kafka/v0/kafka_snap.py

```diff
@@ -88,7 +88,7 @@ def __init__(self) -> None:
         self.snap_config_path = SNAP_CONFIG_PATH
         self.kafka = snap.SnapCache()["kafka"]
 
-    def install(self) -> bool:
+    def install(self, channel) -> bool:
```

Comment:
Also can all these `kafka_snap.py` file changes be made to the `kafka-operator` repo, and not here?
It will need a bump to patch version on L55, which I believe is on `LIBPATCH=4` currently.

If you add that there (and I guess the same code for Kafka VM), by the time it merges and is published, methods that changed from `LIBPATCH 2 -> 4` will have been merged in https://github.com/canonical/zookeeper-operator/pull/21, saving a refactor for this change.